<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thired</class>
 <widget class="QWidget" name="thired">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QRadioButton" name="radioread_2">
         <property name="text">
          <string>读取</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>180</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="radiowrite_2">
         <property name="text">
          <string>写入</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>从机地址:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="machineaddress_2">
         <property name="cursorPosition">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>寄存器起始地址：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="address_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>操作寄存器个数）</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addresslenth_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>如果是写入则输入写入值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="jicunvalue_2">
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="subbut_2">
         <property name="text">
          <string>submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitbut_2">
         <property name="text">
          <string>exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitbut_2</sender>
   <signal>clicked()</signal>
   <receiver>thired</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
